test/utility.test.js

```diff
@@ -702,6 +702,33 @@ describe('scrub', function() {
     expect(result.inner.a).to.be.ok();
     expect(result.inner.b).to.eql('yes');
   });
+  it('should handle scrubPaths', function() {
+    var data = {
+      a: {
+        b: {
+          foo: 'secret',
+          bar: 'stuff'
+        },
+        c: 'bork',
+        password: 'abc123'
+      },
+      secret: 'blahblah'
+    };
+    var scrubPaths = [
+      'nowhere', // path not found
+      'a.b.foo', // nested path
+      'a.password', // nested path
+      'secret' // root path
+    ];
+
+    var result = _.scrub(data, [], scrubPaths);
+
+    expect(result.a.b.bar).to.eql('stuff');
+    expect(result.a.b.foo).to.not.eql('secret');
```

Comment:
a nitpick, but maybe a better check of the actual redaction would be to see if it's
```
expect(result.a.b.foo).to.eql('*******');
```

Reply:
Good call. I had reused the same pattern from other tests, assuming the `redact()` function isn't available here. Actually it is, so I've updated all existing tests as well.